### PR TITLE
Organized Card stories structure

### PR DESCRIPTION
**Pull request by sanji11**
#### What does this PR do?
Organized the `Card/stories` by type of its component which is Layout.

#### Where should the reviewer start?
`src/js/components/Card/stories/Card.stories.js`

#### What testing has been done on this PR?

- `yarn storybook`
- `yarn test`
- `yarn prettier`
- `yarn lint`

#### How should this be manually tested?
`yarn storybook`

#### Any background context you want to provide?
No

#### What are the relevant issues?
#4651 

#### Screenshots (if appropriate)
image

#### Do the grommet docs need to be updated?
No

#### Should this PR be mentioned in the release notes?
No
#### Is this change backwards compatible or is it a breaking change?
This change is backwards compatible.

### CODE REVIEW

File: src/js/components/Card/stories/RichFooter.js [hunk 1]
```diff
@@ -99,5 +98,3 @@ const Example = () => {
     </Grommet>
   );
 };
```
Comment (shabtaisharon) [inline src/js/components/Card/stories/RichFooter.js:100]: In order to not avoid changes to the name of the story you can use the story name parameter
```suggestion
};

RichFooter.story = { name: 'Rich footer' };
```

**Comment by ShimiSun** [comment]
Congrats @sanji11 on your first-time contribution to grommet! Thank you for your contribution, feel free to make more contributions to help our open-source!

**Review by ShimiSun — APPROVED** [review]

**Comment by sanji11** [comment]
Thanks a lot for accepting my PR 😃 I am hoping to contribute more on this following issue. Also, thanks @shabtaisharon for your review. I will keep in mind on my future PR.

**Comment by ShimiSun** [comment]
@sanji11 great to have you on board! 
I'm not sure I've seen you on our grommet-slack, feel free to join us, and chat https://slack-invite.grommet.io.
How did you find this issue?

**Comment by sanji11** [comment]
Sure, I will join on the slack 😄 
After attempting the first PR, I feel comfortable. Now, I am working on Box components. Will send PR soon:)

**Pull request merged**